rerun-failed-run/action.yml

```diff
@@ -51,4 +51,25 @@ runs:
       shell: bash
       run: |
         echo ${{ steps.github-token.outputs.token }} | gh auth login --with-token
-        gh workflow run rerun-failed-run.yml -R camunda/infra-global-github-actions --ref=main -F run_id=${{ inputs.run-id }} -F repository=${{ inputs.repository }} -F error_message=${{ inputs.error-message }}
+
+        multiline_variable=$(cat << EOF
+        Process completed with exit code 88
+        Process completed, with exit code 99
+        EOF
+        )
+        json_array=()
+        counter=1
+        # Read each line of the multiline string
+        IFS=$'\n'
+        for line in $multiline_variable
+        do
+           json_array+="\"error$counter\": \"$line\", "
```

Comment:
I don't really see why produce a map here (`{"error1": "...", "error2": ...}`) if it's much easier to make a flat list of strings (`["...", "..."]`), like this:

```bash
multiline_variable=$(cat << EOF
Process completed with exit code 88
Process completed, with exit code 99
EOF
)
result="["

for line in $multiline_variable
do
  result+="\"$line\","
done

result=$(echo $result | sed '$ s/,$//g')
result+="]"

echo $result
```

Reply:
yeah this actually makes much more sense, will address this